README.md

```diff
@@ -124,8 +124,11 @@ We also evaluate our trained models on four robustness datasets: ImageNetV2/A/R/
 
 Here we provide four jupyter notebooks which contain performance visualization such as GradCAM images (for checkpoint models), superclass performance, model cascades and oracle upper bound. Please refer to detailed documentation [here](model_analysis/README.md).  
 
-## [Retrieval Performance](retrieval/)
-We carry out image retrieval on ImageNet-1K, ImageNetV2 and ImageNet-4K, which we created as an out-of-distribution dataset. A detailed description of the retrieval pipeline is provided [here](retrieval/README.md).
+## [Retrieval](retrieval/)
+We carry out image retrieval on ImageNet-1K, ImageNetV2 and ImageNet-4K, which we created as an out-of-distribution dataset. A detailed description of the retrieval pipeline is provided [here](retrieval/README.md). 
```

Comment:
Just say ImageNet-1K and ImageNet-4K with respective query sets (say V1 and V2 for 1K and 4K query set). Break the sentence and say ImageNet-4K in the next sentence. 

Rest looks good.